roles/docker_node/tasks/docker-ubuntu.yml

```
@@ -0,0 +1,81 @@
+- name: Install prerequisites
+  apt:
+    name:
+      - ca-certificates
+      - curl
+      - jq
+    state: present
+    update_cache: yes
+  become: true
+
+- name: Create keyrings directory
+  file:
+    path: /etc/apt/keyrings
+    state: directory
+    mode: "0755"
+
+- name: Add Docker GPG key
+  shell: curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+  args:
+    creates: /etc/apt/keyrings/docker.asc
+  become: true
+
+- name: Set permissions on GPG key
+  file:
+    path: /etc/apt/keyrings/docker.asc
+    mode: "0644"
+  become: true
+  
+- name: Add Docker repository
+  apt_repository:
+    repo: "deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu {{ ansible_distribution_release }} stable"
+    state: present
+    filename: docker
+  become: true
+
+- name: Update apt cache
+  apt:
+    update_cache: yes
+  become: true
+
+- name: Install Docker packages
+  apt:
+    name:
+      - docker-ce
+      - docker-ce-cli
+      - containerd.io
+      - docker-buildx-plugin
+      - docker-compose-plugin
```

Should we use the same variables here that we do for CentOS instead of hard coding these? Are the package names different between operating systems? Should we add variables for the plugins to follow this precedent? I'm almost wondering if we should just have a single list variable instead of different variables for each package?

Example variable declaration
```
docker_packages:
  - docker-ce
  - docker-ce-cli
  - containerd.io
  - docker-buildx-plugin
  - docker-compose-plugin
```

```suggestion
    - "{{ docker_packages }}"
```